bin/init.sh

```diff
@@ -62,8 +62,9 @@ cp -a "${ISTIO_OUT}/release/istioctl-linux-amd64" /gobin/istioctl
 popd > /dev/null
 
 # Copy install/samples files over from Istio. These are needed by the tests.
-rm -rf "${ISTIOIO_GO}/samples" "${ISTIOIO_GO}/tests/integration" "${ISTIOIO_GO}/manifests"
+rm -rf "${ISTIOIO_GO}/samples" "${ISTIOIO_GO}/tools" "${ISTIOIO_GO}/tests/integration" "${ISTIOIO_GO}/manifests"
 cp -a "${ISTIO_GO}/samples" "${ISTIOIO_GO}/samples"
+cp -a "${ISTIO_GO}/tools" "${ISTIOIO_GO}/tools"
```

Comment:
This is pulling a lot of files. (~60) - even [perhaps] unrelated ones like istio-iptables, packaging, bug-report etc! Is this needed or should this be narrowed down to `${ISTIO_GO}/tools/certs`?